api-2.7/src/main/java/org/openmrs/module/initializer/api/conceptreferencerange/ConceptReferenceRangeLineProcessor.java

```diff
@@ -0,0 +1,50 @@
+package org.openmrs.module.initializer.api.conceptreferencerange;
+
+import org.openmrs.api.ConceptService;
+import org.openmrs.module.initializer.api.BaseLineProcessor;
+import org.openmrs.module.initializer.api.CsvLine;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import org.openmrs.ConceptReferenceRange;
+
+@Component
+public class ConceptReferenceRangeLineProcessor extends BaseLineProcessor<ConceptReferenceRange> {
+	
+	private final String HEADER_CONCEPT_NUMERIC_UUID = "Concept Numeric UUID";
```

Comment:
And so on throughout. I think this is more consistent with how things are done:

```suggestion
	private final String HEADER_CONCEPT_NUMERIC_UUID = "Concept numeric uuid";
```